### PR TITLE
[4.0] Add update scripts for manifest file back end move

### PR DESCRIPTION
**Pull request by laoneo**
Adds the missing update scripts for the back end move of #21790, fixed with 024d943c104f6a489403d42fcdf12590960a6673.

### CODE REVIEW

File: administrator/components/com_admin/sql/updates/mysql/4.0.0-2018-09-12.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+UPDATE `j_extensions` SET `client_id` = 1 WHERE `name` = 'com_mailto';
```
Comment (brianteeman) [inline administrator/components/com_admin/sql/updates/mysql/4.0.0-2018-09-12.sql]: j_ ???

Comment (laoneo) [reply administrator/components/com_admin/sql/updates/mysql/4.0.0-2018-09-12.sql]: Thanks for the hint. I tested it locally and committed with my local prefix. Fixed.

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 0707dcf37321434ea5b8885dde428c79eb892e1f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22138">issues.joomla.org/tracker/joomla-cms/22138</a>.</sub>

**Comment by wilsonge** [comment]
Thanks guys!

**Pull request merged**